src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -114,40 +114,26 @@ function MoneyRequestParticipantsSelector({
             };
         }
 
-        const optionList = OptionsListUtils.getFilteredOptions(
-            options.reports,
-            options.personalDetails,
+        const optionList = OptionsListUtils.getFilteredOptions({
+            reports: options.reports,
+            personalDetails: options.personalDetails,
             betas,
-            '',
-            participants as Participant[],
-            CONST.EXPENSIFY_EMAILS,
+            selectedOptions: participants as Participant[],
+            excludeLogins: CONST.EXPENSIFY_EMAILS,
 
             // If we are using this component in the "Submit expense" or the combined submit/track flow then we pass the includeOwnedWorkspaceChats argument so that the current user
             // sees the option to submit an expense from their admin on their own Workspace Chat.
-            (iouType === CONST.IOU.TYPE.SUBMIT || iouType === CONST.IOU.TYPE.CREATE || iouType === CONST.IOU.TYPE.SPLIT) && action !== CONST.IOU.ACTION.SUBMIT,
+            includeOwnedWorkspaceChats: (iouType === CONST.IOU.TYPE.SUBMIT || iouType === CONST.IOU.TYPE.CREATE || iouType === CONST.IOU.TYPE.SPLIT) && action !== CONST.IOU.ACTION.SUBMIT,
 
             // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-            (canUseP2PDistanceRequests || iouRequestType !== CONST.IOU.REQUEST_TYPE.DISTANCE) && !isCategorizeOrShareAction,
-            false,
-            {},
-            [],
-            false,
-            {},
-            [],
+            includeP2P: (canUseP2PDistanceRequests || iouRequestType !== CONST.IOU.REQUEST_TYPE.DISTANCE) && !isCategorizeOrShareAction,
             // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-            (canUseP2PDistanceRequests || iouRequestType !== CONST.IOU.REQUEST_TYPE.DISTANCE) && !isCategorizeOrShareAction,
-            false,
-            false,
-            0,
```

Comment:
`maxRecentReportsToShow: 0` is missing

Reply:
@c3024 Btw this is crucial missing 😂

Reply:
Yea, my bad!

Reply:
Done.


https://github.com/user-attachments/assets/9d9ae265-8856-4734-872c-b262111dfe72